google/cloud/sql/connector/connector.py

```diff
@@ -104,3 +104,35 @@ def get_ephemeral(service, project, instance, pub_key):
     response = request.execute()
 
     return response["cert"]
+
+
+def connect(instance_connection_string, connection_args=None):
```

Comment:
Would `kwargs` be a better fit here?

Reply:
I think `kwargs` would probably be a better fit, it allows for more flexibility on the developer's end. Also helps make it more clear what is being passed in.